src/string_tensor_pack.cpp

```diff
@@ -1,33 +0,0 @@
-// Copyright (C) 2018-2025 Intel Corporation
```

Comment:
Now you remove cpp, we may expect problems on Mac.
Let us do not touch impplementations of these ext. ops

cc, @ilya-lavrenov